src/base/static/js/views/app-view.js

```diff
@@ -347,6 +348,20 @@ export default Backbone.View.extend({
         return [...collection.models.map(model => model.toJSON()), ...places];
       }, []);
       store.dispatch(setPlaces(allPlaces));
+
+      // Update the load statuses of our layers that are being fetched:
+      const layers = mapLayerStatusesSelector(store.getState());
+      Object.values(layers).forEach(layerStatus => {
+        if (layerStatus.loadStatus === "fetching") {
+          if (layerStatus.isVisible) {
+            // mark the layer to be loaded onto the map
+            store.dispatch(setLayerLoading(layerStatus.id));
+          } else {
+            // the layer is now 'fetched', but don't load it into the map.
+            store.dispatch(setLayerUnloaded(layerStatus.id));
+          }
+        }
+      });
```

Comment:
Instead of updating the load status in mapseed-api-client, we are now doing it here, after the places are set in the redux store.

Also, if the place is not visible, we are updating the load status from "fetching" -> "unloaded". This would occur when the place is not visible by default.